### PR TITLE
Added common page extension and integrated with (old) menu

### PR DESCRIPTION
**Pull request by Bartvaderkin**
This supports filtering menu on requires auth/BSN, setting a number-indicator lookup and setting a menu icon.

### CODE REVIEW

File: src/open_inwoner/cms/extensions/cms_menus.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+from menus.menu_pool import menu_pool
+
+from open_inwoner.accounts.choices import LoginTypeChoices
+from open_inwoner.cms.extensions.constants import IndicatorChoices
+from open_inwoner.cms.extensions.models import CommonExtension
+
+
+def lookup_plan_contacts(request, namespace) -> int:
+    if request.user.is_authenticated:
+        return request.user.get_plan_contact_new_count() or 123
```
Comment (Bartvaderkin) [inline src/open_inwoner/cms/extensions/cms_menus.py:17]: `or 123` ? :wink:

File: src/open_inwoner/cms/extensions/cms_menus.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import random
```
Comment (vaszig) [inline src/open_inwoner/cms/extensions/cms_menus.py:1]: These two lines can be removed as well, I don't see them used somewhere

**Review by vaszig — APPROVED** [review]
Besides the small remark about imports and the numbers you already mentioned, it looks nice!

**Review by vaszig — APPROVED** [review]
It looks nice!My question is don't we need an icon for the profile as well?

**Comment by Bartvaderkin** [comment]
Probably, but these are temporary icons, @jiromaykin might replace them.

**Pull request merged**